app/uk/gov/hmrc/fhregistrationfrontend/views/enrolment_pending.scala.html

```diff
@@ -15,47 +15,57 @@
  *@
 
 @import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.layout
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
 
-@this(
-    mainTemplate: uk.gov.hmrc.fhregistrationfrontend.views.html.main_template_old
-)
+@this(layout: layout, viewHelpers: ViewHelpers)
 
 @()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
-@stylesheets = {<link rel="stylesheet" href='@routes.Assets.at("stylesheets/status.css")'/>}
 
-@title = @{
-  Messages(s"fh.status.received.title")
+@title1 = @{
+  Messages("fh.status.received.title")
 }
 
-@mainTemplate(title = title, linkElement = Some(stylesheets)) {
+@layout(title = title1) {
 
-  <div class="govuk-box-highlight appstatus">
-    <h1 class="heading-xlarge">
-      @title
-    </h1>
-  </div>
+    @viewHelpers.govukPanel(Panel(
+        title = Text(Messages("fh.status.received.title")),
+        content = HtmlContent("")
+    ))
 
-  <h2 class="heading-large">
+    <h2 class="govuk-heading-l">
     @Messages("fh.status_pending.what_happens_next")
-  </h2>
+    </h2>
+    <div class="govuk-body">
+      <p>
+        @Messages("fh.status_pending.what_happens_next_1")
+      </p>
 
-  <p>
-    @Messages("fh.status_pending.what_happens_next_1")
-  </p>
-
-  <p>
-    @Messages("fh.status_pending.what_happens_next_2")
-  </p>
+      <p>
+        @Messages("fh.status_pending.what_happens_next_2")
+      </p>
 
+      <p>
+        @Messages("fh.status_pending.what_happens_next_3")
+        <a href=@{Messages("link.vat")} target="_blank">@Messages("fh.status_pending.what_happens_next_4")</a>
+        @Messages("fh.status_pending.what_happens_next_5")
+      </p>
+    </div>
   <p>
-    @Messages("fh.status_pending.what_happens_next_3")
-    <a href=@{Messages("link.vat")} target="_blank">@Messages("fh.status_pending.what_happens_next_4")</a>
-    @Messages("fh.status_pending.what_happens_next_5")
-  </p>
-
-  <p>
-    <a class="button" role="button" href="@uk.gov.hmrc.fhregistrationfrontend.controllers.routes.SignOutController.signout">
-    @Messages("fh.status_pending.finish")
-    </a>
+      <a href="@uk.gov.hmrc.fhregistrationfrontend.controllers.routes.SignOutController.signout" role="button"
```

Comment:
This has a class of button because of the way it is supposed to look on the page - there is no link on the button, but the button acts like a link.  Class govuk-link not added.

Reply:
If it's button, then you should use button component not <a> tag